api/src/server.js

```diff
@@ -0,0 +1,8 @@
+import api from "./api";
+
+/* API listen port */
+const PORT = process.env.PORT || 3000;
+
+api.listen(PORT, () => {
+  console.log(`Adelphi API listening at http://localhost:${PORT}`)
```

Comment:
The same way we access the results server right now with `kubectl port-foward`.
So if you hit the /api/ path, it will proxy to the node server internally:

https://github.com/datastax/adelphi/pull/167/files#diff-9d4b4d1d3a7b9885d6a08c0673d7bfdf12d031ceb458c08038c0cbdc32f21951R17